schedule_lessons/dashboard/static/dashboard/js/agenda.js

```diff
@@ -60,7 +64,7 @@ function getCookie(name) {
     for (var i = 0; i < cookies.length; i++) {
       var cookie = jQuery.trim(cookies[i]);
       // Does this cookie string begin with the name we want?
-      if (cookie.substring(0, name.length + 1) == (name + '=')) {
+      if (cookie.substring(0, name.length + 1) == name + '=') {
```

Comment:
wish we were using lodash to do stuff like this, maybe take a look and see if you can include it - otherwise, all good